### PR TITLE
Update AfterBurner version to match other Jackson versions

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/pom.xml [hunk 1]
```diff
@@ -239,7 +239,7 @@ Licensed under the MIT License.
                 <include>com.fasterxml.jackson.core:jackson-annotations:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-annotations;external_dependency} -->
                 <include>com.fasterxml.jackson.core:jackson-databind:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
                 <include>com.fasterxml.jackson.datatype:jackson-datatype-jsr310:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.datatype:jackson-datatype-jsr310;external_dependency} -->
-                <include>com.fasterxml.jackson.module:jackson-module-afterburner:[2.10.1]</include> <!-- {x-include-update;cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
+                <include>com.fasterxml.jackson.module:jackson-module-afterburner:[2.11.2]</include> <!-- {x-include-update;cosmos_com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
```
Comment (alzimmermsft) [inline sdk/cosmos/azure-cosmos-encryption/pom.xml:242]: Looks like this was a copy and paste from `azure-cosmos` without this module actually using this dependency.

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-encryption/pom.xml:242]: Yes, we should get rid of it, good catch @alzimmermsft

**Review by kushagraThapar — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos-encryption/pom.xml (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-encryption/pom.xml:242]: Can you please remove it in this PR ?

Comment (alzimmermsft) [reply sdk/cosmos/azure-cosmos-encryption/pom.xml:242]: Will remove it in the next commit

**Pull request merged**